src/shared/components/home/site-form.tsx

```diff
@@ -1208,20 +1207,25 @@ export class SiteForm extends Component<SiteFormProps, SiteFormState> {
     event: InfernoMouseEvent<HTMLButtonElement>
   ) {
     event.preventDefault();
-    let taglines = i.state.siteForm.taglines.unwrap();
-    taglines.splice(index, 1);
-    i.state.siteForm.taglines = Some(taglines);
-    i.setState(i.state);
+    if (i.state.siteForm.taglines.isSome()){
+      let taglines = i.state.siteForm.taglines.unwrap();
+      taglines.splice(index, 1);
+      i.state.siteForm.taglines = None; // force rerender of table rows
+      i.setState(i.state);
+      i.state.siteForm.taglines = Some(taglines);
+      i.setState(i.state);
```

Comment:
No need for two setStates here, unless something really weird is going on.

Reply:
Unfortunately there is. The key for the repeater in the html is 'index', so when I delete a row, its possible that certain rows don't re-render, causing issues with displaying the markdown editor.

Reply:
Gotcha